checker/src/test/java/org/checkerframework/checker/test/junit/NullnessJSpecifyConformanceTest.java

```diff
@@ -0,0 +1,113 @@
+package org.checkerframework.checker.test.junit;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+
+import org.checkerframework.checker.nullness.NullnessChecker;
+import org.checkerframework.framework.test.TestConfiguration;
+import org.checkerframework.framework.test.TestConfigurationBuilder;
+import org.checkerframework.framework.test.TestUtilities;
+import org.checkerframework.framework.test.TypecheckExecutor;
+import org.checkerframework.framework.test.TypecheckResult;
+import org.jspecify.conformance.ConformanceTestRunner;
+import org.jspecify.conformance.ReportedFact;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/** An object to run the conformance tests against the EISOP Checker Framework. */
+public final class NullnessJSpecifyConformanceTest {
+
+    private final Path testDir;
```

Comment:
The CI will complain about missing javadoc. Please add some description to all fields and methods.